tf2_gnn/cli_utils/model_utils.py

```diff
@@ -26,6 +31,7 @@ def save_model(save_file: str, model: GraphTaskModel, dataset: GraphDataset) ->
         "num_edge_types": dataset.num_edge_types,
         "node_feature_shape": dataset.node_feature_shape,
     }
+    data_to_store.update(extra_data_to_store)
```

Comment:
Just noting a small consequence of this: one can (accidentally or not) overwrite pre-existing entries in `data_to_store`.

Reply:
Yup, I was debating in what order I should do things, but then settled on this because it gives a way to override default saving behaviour if that is desired...